config

```diff
@@ -75,12 +76,17 @@ else
   exit 1
 fi
 
-if [ "$NGX_DEBUG" = "YES" ]; then
-  buildtype=Debug
-  # If we're using a psol tarball that doesn't contain Debug/ (which is the case
-  # from 1.12 onward) then this will be overriden to buildtype=Release below.
+psol_buildtype="${PSOL_BUILDTYPE:-unset}"
```

Comment:
One request -- can you move this to the top where the others are/declared initialized?
(maybe below `position_aux="${POSITION_AUX:-unset}"`)?

Reply:
sure 😃